gravitee-apim-console-webui/src/management/integrations/integration-overview/integration-overview.component.ts

```diff
@@ -56,14 +53,53 @@ export class IntegrationOverviewComponent implements OnInit {
     private readonly activatedRoute: ActivatedRoute,
     private readonly router: Router,
     private readonly snackBarService: SnackBarService,
-    private readonly matDialog: MatDialog,
   ) {}
 
   ngOnInit(): void {
+    if (this.integrationsService.isIngestToRun()) {
+      this.ingest();
+    }
     this.getIntegration();
     this.initFederatedAPIsList();
   }
 
+  private ingest() {
+    const { integrationId } = this.activatedRoute.snapshot.params;
+    this.isIngesting = true;
+    this.integrationsService.setIsIngestToRun(false);
```

Comment:
It's not clear for me. We need the boolean? We can't use only one?

Reply:
we use it also in different context: 
```
public proceedIngest() {
    this.integrationsService.setIsIngestToRun(true);
```